vips/vip-492/bsctestnet.ts

```diff
@@ -0,0 +1,139 @@
+export const BOUND_VALIDATOR_OP_SEPOLIA = "0x482469F1DA6Ec736cacF6361Ec41621f811A6800";
+export const DEFAULT_PROXY_ADMIN_OP_SEPOLIA = "0xa9aaf2A1cCf2C3a87997942abaA740887cC89241";
+export const RESILIENT_ORACLE_IMPLEMENTATION_OP_SEPOLIA = "0xe36F76dc26885CcEce97B96f80f4FA58c89772Fc";
+export const CHAINLINK_ORACLE_IMPLEMENTATION_OP_SEPOLIA = "0x15242a55Ad1842A1aEa09c59cf8366bD2f3CE9B4";
+export const BOUND_VALIDATOR_IMPLEMENTATION_OP_SEPOLIA = "0xca8c824E577e1E2EDF4442cB46046ab000FE76CF";
+
+export const RESILIENT_ORACLE_UNICHAIN_SEPOLIA = "0xA469E718BDE2C9939bD29529A38184e97dF0A741";
+export const REDSTONE_ORACLE_UNICHAIN_SEPOLIA = "0x8683D6902A669Ac479cfcCf2542724a133D4d872";
+export const BOUND_VALIDATOR_UNICHAIN_SEPOLIA = "0x51C9F57Ffc0A4dD6d135aa3b856571F5A4e4C6CB";
+export const DEFAULT_PROXY_ADMIN_UNICHAIN_SEPOLIA = "0x256735eFdfDf135bD6991854e0065909e57804aa";
+export const RESILIENT_ORACLE_IMPLEMENTATION_UNICHAIN_SEPOLIA = "0x4E953e3741a17aFaD69776742d1ED1c0130F43f7";
+export const REDSTONE_ORACLE_IMPLEMENTATION_UNICHAIN_SEPOLIA = "0x44A47AfC1A9467Dfe1D5E967cA78432C699a13d9";
+export const BOUND_VALIDATOR_IMPLEMENTATION_UNICHAIN_SEPOLIA = "0x15242a55Ad1842A1aEa09c59cf8366bD2f3CE9B4";
+
+export const vip492 = () => {
+  const meta = {
+    version: "v2",
+    title: "",
+    description: ``,
+    forDescription: "Execute this proposal",
+    againstDescription: "Do not execute this proposal",
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: DEFAULT_PROXY_ADMIN_BASE_SEPOLIA,
+        signature: "upgrade(address,address)",
+        params: [RESILIENT_ORACLE_BASE_SEPOLIA, RESILIENT_ORACLE_IMPLEMENTATION_BASE_SEPOLIA],
+        dstChainId: LzChainId.basesepolia,
+      },
+      {
+        target: DEFAULT_PROXY_ADMIN_BASE_SEPOLIA,
+        signature: "upgrade(address,address)",
+        params: [CHAINLINK_ORACLE_ORACLE_BASE_SEPOLIA, CHAINLINK_ORACLE_IMPLEMENTATION_BASE_SEPOLIA],
+        dstChainId: LzChainId.basesepolia,
+      },
+      {
+        target: DEFAULT_PROXY_ADMIN_BASE_SEPOLIA,
+        signature: "upgrade(address,address)",
+        params: [REDSTONE_ORACLE_ORACLE_BASE_SEPOLIA, REDSTONE_ORACLE_IMPLEMENTATION_BASE_SEPOLIA],
+        dstChainId: LzChainId.basesepolia,
+      },
+      {
+        target: DEFAULT_PROXY_ADMIN_BASE_SEPOLIA,
+        signature: "upgrade(address,address)",
+        params: [BOUND_VALIDATOR_BASE_SEPOLIA, BOUND_VALIDATOR_IMPLEMENTATION_BASE_SEPOLIA],
+        dstChainId: LzChainId.basesepolia,
+      },
+      {
+        target: RESILIENT_ORACLE_BASE_SEPOLIA,
+        signature: "setTokenConfig((address,address[3],bool[3],bool))",
+        params: [
+          [
+            wSuperOETHb,
+            [wSuperOETHb_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+            false,
+          ],
+        ],
+        dstChainId: LzChainId.basesepolia,
+      },
```

Comment:
Let's set the snapshot gap and the snapshot interval (30 days), here and for basemainnet. The snapshot gap should be 1.11% of the `snapshotMaxExchangeRate`